libraries/HTTPClient/src/HTTPClient.cpp

```diff
@@ -1166,6 +1166,7 @@ bool HTTPClient::connect(void)
 
     // set Timeout for WiFiClient and for Stream::readBytesUntil() and Stream::readStringUntil()
     _client->setTimeout(_tcpTimeout);	
+    _client->setConnectionTimeout(_tcpTimeout);
```

Comment:
This won't be necessary as it will set just the _timeout variable if changed according to my comment the same as the `setTimeout` function.

Reply:
setTimeout is Stream:setTimeout and sets the timeout field in Stream. setConnectionTimeout setts the timeout filed in WiFiClient

Reply:
I see